src/main/webapp/js/afterGame.js

```diff
@@ -7,8 +10,75 @@ function loadGameData() {
   var request = new Request('/load-game-data', {method: 'POST', body: fetchParams});
   fetch(request).then(response => response.json()).then(async (data) => {
     var gameTitle = document.getElementById('game-title');
-    gameTitle.innerHTML = '<h1>' + data.gameName + '</h1>';
-
-    var difficulty = document.getElementById('difficulty');
+    gameTitle.innerHTML = '<h1>' + data.gameName + '</h1>';    
   });
-}
\ No newline at end of file
+}
+
+/** 
+* Creates a row of the number of stars the user clicked on and places then at className 'stars'
+* @param {int} numOfStars the number of stars the user clicked on
+*/
+function stars(numOfStars) {  
+  var startOfStar = '<i class="large material-icons" onclick="stars(';
+  var fullStar = ')">star</i>';
+  var halfStar = ')">star_half</i>';
+  var emptyStar = ')">star_border</i>';
+
+  var count = 0;
+  var fiveStars = '';
+  for (var i = 0; i < 5; i++) {
+    if(numOfStars >= 1.0) {
```

Comment:
Maybe rename to 'starsRemaining'

Reply:
I can understand why you would say that, but I also feel that would cause confusion since it is the variable being passed in and the name in the JS doc. I feel numOfStars clearly indicates that the number of stars being passed in the the amount of stars that there should be and even though it is being decreased, it still is the numOfStars as well.